recipes/softsv/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "SoftSV" %}
+{% set version = "1.4.2" %}
+{% set sha256hash = "b3caea962569f7c30f1257ad148fd1e833f8d0737105b9983579d91da2bed6e8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/softsv/SoftSV_{{ version }}.zip
+  sha256: {{ sha256hash }}
+  patches:
+    - makefile.diff
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - bamtools ==2.4.1
+    - boost ==1.62.0
+    - seqan ==2.1.1
+  run:
+    - bamtools ==2.4.1
+    - boost ==1.62.0
+    - seqan ==2.1.1
+
+test:
+  commands:
+    - SoftSV --help
+
+about:
+  home: https://sourceforge.net/projects/softsv
+  license: GPLv3
+  summary: 'SoftSV is a tool for the detection of small and large deletions, inversions, tandem duplications and translocations from paired-end sequencing data.'
```

Comment:
Try to keep this to one line. It's shown on `anaconda.org/bioconda/`*`packagename`* in a place where more than a line looks weird. It's the title:

```
summary: Find deletions, inversions and tandem duplications in paired end reads
description: |
  SoftSV is a tool for the detection of small and large deletions, inversions, tandem
  duplications and translocations from paired-end sequencing data.

  Lore ipsum
 
  more ipsum
```    

(While the description field is an entire pane at the bottom where you can put lots of googleable text)